source/fundamentals/versioned-api.txt

```diff
@@ -0,0 +1,128 @@
+=============
+Versioned API
+=============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. note::
+
+   The Versioned API feature requires MongoDB Server 5.0 or later.
+
+   You should only use the Versioned API feature if all the MongoDB
+   servers you are connecting to support this feature.
+
+This guide shows you how to specify the **Versioned API** when connecting to
+a MongoDB instance or replica set. You can use the Versioned API feature to
+force the server to run operations with behavior compatible with the
+specified **API version**. An API version defines the expected behavior of the
+operations it covers and the format of server responses. If you change to
+a different API version, the operations are not guaranteed to be
+compatible and the server responses are not guaranteed to be similar.
+
+When you use the Versioned API feature with an official MongoDB driver, you
+can update your driver or server without worrying about backward compatibility
+issues of the commands covered by the Versioned API.
+
+See the server manual page on `Versioned API <https://docs.mongodb.com/v5.0/reference/versioned-api/>`__
+for more information including a list of commands it covers.
+
+The following sections describe how you can enable Versioned API for
+your MongoDB client and the options that you can specify.
+
+Enable Versioned API on a MongoDB Client
+----------------------------------------
+
+To enable the Versioned API, you must specify an API version in the settings
+of your MongoDB client. Once you instantiate a ``MongoClient`` instance with
+a specified Versioned API, all commands you run with that client use that
+version of the Versioned API. If you need to run commands not covered by the
+Versioned API or with a different version of it, instantiate a separate client.
```

Comment:
> If you need to run commands not covered by theVersioned API 

You can just leave apiStrict off for this case

Reply:
I'm still a bit confused by this.  There are two paragraphs under the tip, both of which are correct.  But the one not under the tip repeats parts of both tip paragraphs, and offers slightly different advice.

I think this paragraph could just be removed?

Reply:
Sorry, forgot to remove this after moving it to the tip section.